lib/rules/eqeqeq.js

```diff
@@ -98,8 +98,9 @@ module.exports = {
          * @private
          */
         function isNullCheck(node) {
-            return (node.right.type === "Literal" && node.right.value === null) ||
```

Comment:
the linter flagged it. If everything else looks good then I can follow up on that and fix this case.